.github/workflows/moq-cli.yml

```
@@ -130,12 +135,11 @@ jobs:
       matrix:
         include:
-          # Both Apple targets build on the Apple Silicon runner. The
-          # Determinate Nix installer dropped Intel macOS, so x86_64 is
-          # cross-compiled (Apple clang is multi-arch) and its smoke test
-          # runs under Rosetta. See rs/scripts/package-binary.sh.
+          # Each Apple arch builds natively on its own runner (macos-13 is the
+          # last Intel image; macos-latest is Apple Silicon) so capture links a
+          # matching-arch Homebrew ffmpeg. No cross-compile or Rosetta needed.
           - target: x86_64-apple-darwin
-            os: macos-latest
+            os: macos-13
```

actionlint (1.7.12)

[error] label "macos-13" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)